src/openMVG/multiview/CMakeLists.txt

```diff
@@ -32,7 +32,6 @@ target_link_libraries(openMVG_multiview
   LINK_PRIVATE
     openMVG_graph
     ${CERES_LIBRARIES}
-    ${OPENMVG_LIBRARY_DEPENDENCIES}
```

Comment:
We should keep this, it was introduced in a97bf6ab09e5783d86e0eda8cf160060b30b99a4 to fix #2036